src/com/rcv/CastVoteRecord.java

```diff
@@ -119,6 +119,12 @@ public void setCurrentRecipientOfVote(String currentRecipientOfVote) {
     this.currentRecipientOfVote = currentRecipientOfVote;
   }
 
+  // function: getPrecinct
+  // purpose: getter for precicnt
+  // returns: value of field
+  public String getPrecinct() {
```

Comment:
What's the advantage of using the getter instead of referencing the instance variable directly? I googled this for a minute and found arguments in both directions. The highlight was definitely from [here](https://www.reddit.com/r/java/comments/2f3flb/is_it_considered_better_practice_for_a_class_to/): "Your analysis is akin to comparing the practice to wearing a condom while masturbating."

Reply:
Wondering why you added it here in the first place? I think the general philosophy is that it's good practice to use getters and setters rather than modifying variables directly [for the reasons outlined here](https://stackoverflow.com/a/1568230/3846321). From personal experience, wanting to add validation later without having to change the references everywhere in your code is really nice (i.e. encapsulation). But since there's not much previous precedent for doing that in this project, maybe don't bother starting until you're actually doing validation of some sort.

Reply:
On the [other hand](https://stackoverflow.com/a/7093579/3846321)... I'd lean towards not using them I guess... either way, consistency is good. I still need to read Effective Java!

Reply:
This is getting divisive.  I'd propose allow direct access for more convenient coding but once we have accessors use them exclusively.  And FWIW, I ALWAYS wear a condom while masturbating.

Reply:
I added the getter here because we need to access this value from Tabulator. I guess my stance is that when we have a getter for an instance variable, it should just be returning the value and not doing anything else anyway. I'm happy to revisit this and change this class and other classes that have a similar issue if we agree on a different approach, but for now will just leave as is for consistency.